README.md

```diff
@@ -13,82 +13,117 @@ It's a debug only package, so it does not affect your production build.
+    [spacejam|meteor] --driver-package practicalmeteor:mocha-console-runner
 
-For app, edit your `package.json` with the following
+If you want to, you can use this syntax, the following two commands are equivalent, but the second one is shorter and thus less typing error-prone
+
+    spacejam test-packages ./ --driver-package practicalmeteor:mocha-console-runner --coverage "out_lcovonly out_html"
+    spacejam-mocha test-packages ./ --coverage "out_lcovonly out_html"
+
+
+
+## Setup spacejam
+
+If you have any trouble, refer to this example of Meteor application [meteor-coverage-app-exemple](https://github.com/serut/meteor-coverage-app-exemple) to see how a test runner can execute yours tests, save coverage and send it to coveralls. Or feel free to open an issue.
+
+
+Add the following dependencies in your `package.json`:
+
+    meteor npm i --save-dev https://github.com/serut/spacejam/tarball/windows-suppport-rc4 
+
+Add what you need to run your app inside your `package.json`:
 
-    {
-    "devDependencies": {
-        [...]
-        "spacejam": "https://github.com/serut/spacejam/tarball/windows-suppport-rc4",
-        "coveralls": "^2.11.11",
-        "codecov.io": "^0.1.6"
-    },
-    {
     "scripts": {
-        [...]
-        "test-coverage-app-unit": "spacejam test                  --driver-package practicalmeteor:mocha-console-runner --coverage out_coverage",
-        "test-coverage-app-full": "spacejam test --full-app       --driver-package practicalmeteor:mocha-console-runner --coverage 'in_coverage|out_lcovonly'",
-        "test-coverage-packages-mocha": "spacejam test-packages   --driver-package practicalmeteor:mocha-console-runner --coverage out_lcovonly"
-    },
-    "devDependencies": {
-        [...]
-        "spacejam": "https://github.com/serut/spacejam/tarball/windows-suppport-rc4",
-        "coveralls": "^2.11.11",
-        "codecov.io": "^0.1.6"
-    },
+        [APP]
+        "test": "meteor npm run lint:fix & meteor npm run test:app-unit-coverage & ...",
+        "test:app-unit-coverage": "spacejam test                  --driver-package practicalmeteor:mocha-console-runner --coverage out_coverage",
+        "test:app-full-coverage": "spacejam test --full-app       --driver-package practicalmeteor:mocha-console-runner --coverage 'in_coverage|out_lcovonly'",
+        "test:packages-coverage": "spacejam test-packages   --driver-package practicalmeteor:mocha-console-runner --coverage out_lcovonly"
+        
+        [PCKGS]
+        "test": "spacejam test-packages ./ --coverage out_lcovonly  --driver-package practicalmeteor:mocha-console-runner",
```

Comment:
{Install `spacejam` as a globally available NPM package and execute `spacejam test...`} OR {install it as a local NPM dev-dependency and execute `node_modules/.bin/spacejam test...` or `node_modules/spacejam/bin/spacejam test...`}, but not mix both :wink: 

Otherwise, users will have to add the binaries path to their `$PATH` environment variable in some way, like `export PATH="$PATH:./node_modules/.bin"` or `export PATH="$PATH:./node_modules/spacejam/bin"` to get the `package.json` script commands working.